src/Dev/Install/client/js/install.js

```diff
@@ -1,21 +1,9 @@
 $(document).ready(function () {
 
     /**
-     * Toggle field readonly modes, if check configuration comes from
-     * environment variables (values populated on reload).
-     */
-    $('#use_environment').click(function (e) {
-        if (!$(this).is(':checked')) {
-            $('.configured-by-env').removeAttr('disabled');
-        } else {
-            $('.configured-by-env').attr('disabled', 'disabled');
-        }
-    });
-
-    /**
      * Hide all existing database warnings, and show only current one
      */
-    $('#database_selection li label, #database_selection input:radio').click(function (e) {
+    $('#database_selection li label, #database_selection input:radio').click(function () {
```

Comment:
could we change the selector to this?
`$('#database_selection li>label, #database_selection li>input:radio')`

because otherwise you can hide all the db fields by clicking the labels for the dbfields